test/dpul_collections_web/live/item_live_test.exs

```diff
@@ -76,7 +76,7 @@ defmodule DpulCollectionsWeb.ItemLiveTest do
     conn =
       get(
         conn,
-        "/i/%D9%83%D9%88-%D8%A7%D8%B4%D9%83%D9%88%DA%BA-%D8%B3%DB%92-%D8%A8%D8%AC%D9%87%D8%A7%D9%86%D8%A7-%D9%87%D9%88%DA%AF%D8%A7/item/3"
+        "http://127.0.0.1:4000/i/%D8%A7%D8%A8-%D9%83%D9%88%D8%A6%D9%89-%D8%AC%D9%86%DA%AF-%D9%86%D9%87-%D9%87%D9%88%DA%AF%D9%89/item/3"
```

Comment:
What's the story here? Does it work if I don't have the phoenix server running in dev?

Reply:
accident. Give me a sec.